openapi_pydantic/v3/v3_0_3/datatype.py

```diff
@@ -4,6 +4,7 @@
 class DataType(str, enum.Enum):
     """Data type of an object."""
 
+    NULL = "null"
```

Comment:
`null` isn't supported in v3.0.3 - https://swagger.io/specification/v3/

Reply:
Wow, learning that was a surprise! I added a test that exercised a nullable field. The test revealed that this package was breaking when generating OpenAPI 3.0.3 schemas containing nullable fields. Adding "null" as a data type was an obvious but wrong fix. I reverted that. Instead, I subclassed Pydantic's `GenerateJsonSchema` class to generate `nullable` fields (just for OpenAPI 3.0.3), and now all the tests pass again.